src/JavaSession1/ArrayRepetition.java

```diff
@@ -0,0 +1,59 @@
+package JavaSession1;/*Question 6: There is an array with every element repeated twice except one. Find that element*/
+
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class ArrayRepetition {
+
+    public static void main(String[] args) {
+
+        int[] integerArray;
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Enter the size of Array");
+        int arraySize = sc.nextInt();
+        integerArray = new int[arraySize];
+        insertInArray(integerArray, arraySize);
+        System.out.println("Element Which Is Repeated Once In Array : " + repeatedElementHashMap(integerArray, arraySize));
+
+
+    }
+
+    private static int repeatedElement(int[] integerArray, int arraySize) {
+        Arrays.sort(integerArray);
+        int i = 0;
+        while (i < arraySize) {
+            if (integerArray[i] != integerArray[i + 1]) {
+                return integerArray[i];
+            }
+            i+=2;
+        }
+        return -1;
+    }
+
+    private static int repeatedElementHashMap(int[] integerArray, int arraySize) {
+        HashMap<Integer, Integer> integerMap = new HashMap<>(arraySize);
```

Comment:
Please use name defining the variable => integerMap does not seems to be correct.
I am also not good in method and variable namings but I would suggest you give more attention while defining. It makes code readable.